include/lsst/afw/geom/SeparableXYTransform.h

```diff
@@ -0,0 +1,86 @@
+// -*- lsst-c++ -*-
+
+/* 
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ * 
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the LSST License Statement and 
+ * the GNU General Public License along with this program.  If not, 
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/**
+ * \file 
+ * @brief Class representing a 2D transform for which the pixel
+ * distortions in the x- and y-directions are separable.
+ */
+
+#ifndef LSST_AFW_GEOM_SEPARABLEXYTRANSFORM_H
+#define LSST_AFW_GEOM_SEPARABLEXYTRANSFORM_H
+
+#include "lsst/afw/geom/XYTransform.h"
+
+namespace lsst {
+namespace afw {
+namespace geom {
+
+class Functor;
+
+class SeparableXYTransform : public XYTransform {
```

Comment:
Please add a document string describing the point of this class. It is fairly self-documenting, but still...